src/lib/ThumbnailFactory.js

```diff
@@ -80,16 +86,62 @@ class ThumbnailFactory {
 
     if (!service) return undefined;
 
-    // just bail to a static image, even though sizes might provide something better
-    if (isLevel0ImageProfile(service)) {
-      return ThumbnailFactory.staticImageUrl(resource);
-    }
-
     const aspectRatio = resource.getWidth()
       && resource.getHeight()
       && (resource.getWidth() / resource.getHeight());
 
-    if (maxHeight && maxWidth) {
+    // just bail to a static image, even though sizes might provide something better
```

Comment:
Is this handling the case where an image information response provides a size but the image server does not respond with it?

Reply:
if the manifest is lying about the available sizes?

Reply:
Yes

Reply:
Nope; I don't think we handle that case for any of our thumbnail calculations, before or after this PR. Do we have examples of that in the wild?

Reply:
No I was just trying to better understand what was happening here. But I think I get it now